internal/renderers/femtovg/fonts.rs

```diff
@@ -163,21 +153,15 @@ pub struct FontCache {
     loaded_fonts: HashMap<FontCacheKey, LoadedFont>,
     // for a given fontdb face id, this tells us what we've learned about the script
     // coverage of the font.
-    loaded_font_coverage: HashMap<fontdb::ID, GlyphCoverage>,
+    loaded_font_coverage: HashMap<fontique::FamilyId, GlyphCoverage>,
```

Comment:
Maybe we can remove this hash map in the future with https://github.com/linebender/parley/pull/413/files . (but not in scope of this PR)